### ISSUE TITLE
Qwickie do not handle FilterForm correctly

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bwebb3856/qwickie#30]
```
What steps will reproduce the problem?
1. Create a page with a DataTable

2. Add a FormFilter to the DataTable :
        // create the form used to contain all filter components
    final FilterForm<SuiviDomaine> form = new FilterForm<SuiviDomaine>("filter-form", suivisProvider) {
        private static final long serialVersionUID = 1L;

        @Override
        protected void onSubmit() {
        dataTable.setCurrentPage(0);
        }
    };
    add(form);
        ...
        dataTable.addTopToolbar(new FilterToolbar(dataTable, form, suivisProvider));

3. Add the HTML code of the FormFilter in the page  :

<form wicket:id="filter-form">
       <input type="hidden" name="tracker" wicket:id="focus-tracker"/>
       <table wicket:id="table" class="dataTable"></table>
       <span wicket:id="focus-restore">[call to focus restore script]</span>
    </form>


What is the expected output? What do you see instead?
Qwickie complains about : "wicket:id "focus-restore" not found in java file(s)

What version of the product are you using? On what operating system?
QWickie 0.9.1.201110081103
Eclipse Indigo SR1 64bits
Windows 7 64bits




```

Original issue reported on code.google.com by `tahitian...@gmail.com` on 9 Nov 2011 at 7:25

Attachments:
- [screenshot.png](https://storage.googleapis.com/google-code-attachments/qwickie/issue-30/comment-0/screenshot.png)
- [screenshot2.png](https://storage.googleapis.com/google-code-attachments/qwickie/issue-30/comment-0/screenshot2.png)

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the good bug report!
```

Original comment by `count.negative` on 11 Nov 2011 at 8:39
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
You are welcome.

Qwickie is a great plugin for eclipse users like me!
```

Original comment by `tahitian...@gmail.com` on 12 Nov 2011 at 12:09

**Comment by GoogleCodeExporter** [comment]
```
Added focus-tracker and focus-restore manually to save time when parsing the 
object tree.
Released 0.9.2

And thanks again for providing such detailed error reports.
```

Original comment by `count.negative` on 12 Nov 2011 at 11:48
- Changed state: **Fixed**
